src/lib/DataTable.tsx

```diff
@@ -122,19 +121,29 @@ class TableRow<T> extends React.PureComponent<TableRowProps<T>> {
                     }
 
                     const value = column._getValue(row);
+
+                    let style = column.customStyle({
+                        row,
+                        index,
+                        name: column.name,
+                        header: false,
+                        footer,
+                        headerData,
+                    });
+
+                    // Fixed cell width for resizeable columns for proper content wrap
+                    if (column.resizeable) {
+                        style = style
+                            ? {...style, width: column.width, maxWidth: column.width}
+                            : {width: column.width, maxWidth: column.width};
```

Comment:
Do we realy need to check`style`? It should work fine:
```ts
style = {...style, width: column.width, maxWidth: column.width};
```

Reply:
Didn't know js correctly spreads undefined)